CHANGELOG.md

```diff
@@ -0,0 +1,67 @@
+# Changelog
+## Unreleased
+### Backend
+  - Upgraded `worship-posts-consumer` and `worship-services-sensitive-consumer` [DL-6424]
+
+### Deploy notes
+We are going to get rid of the database so first, make sure you create and store a backup
+```
+# Create the backup
+/data/useful-scripts/virtuoso-backup.sh `docker ps --filter "label=com.docker.compose.project=app-worship-organizations" --filter "label=com.docker.compose.service=triplestore" --format "{{.Names}}"`
+# Store it on a different server
+/data/useful-scripts/remote-backup.sh u425986-sub2@u425986-sub2.your-storagebox.de 23 /root/.ssh/backups_rsa
+```
+
+Then, we proceed
+```
+drc down
+rm -rf data
+```
+Update `docker-compose.override.yml` to remove the config of `worship-services-sensitive-consumer` and replace it by:
+```
+  worship-services-sensitive-consumer:
+    environment:
+      DCR_SYNC_BASE_URL: 'https://loket.lblod.info/' # The endpoint of your choice
```

Comment:
This seems to be the URL for Loket QA? Is that what we want in the deploy notes? The one below this line is the same.

Reply:
Mmh it doesn't matter too much I think, but as the most critical deploy is prod, i'll put prod's URI to be sure!